benchexec/tools/cpachecker.py

```diff
@@ -180,16 +180,40 @@ def option_present(option):
                         f"Unsupported data_model '{data_model}' defined for task '{task}'"
                     )
 
+        if isinstance(task.options, dict) and "witness" in task.options.keys():
+            if not option_present("witness"):
+                possible_witness_files = list(
+                    filter(
+                        lambda x: x.endswith(task.options.get("witness")),
+                        task.input_files_or_empty,
+                    )
+                )
+                assert (
+                    len(possible_witness_files) == 1
+                ), f"Expected exactly one witness file, but found {len(possible_witness_files)}: {possible_witness_files}"
```

Comment:
Given that we produce a proper error message instead of a crash for a different case of unsupported inputs a few lines below, why not do the same here?